addon/components/single-event-objective-list.hbs

```diff
@@ -1,11 +1,24 @@
-<div class="single-event-objective-list">
-  <h2>
-    {{@title}}
+<div
+  class="single-event-objective-list"
+  data-test-single-event-objective-list
+  {{did-insert (set this.isExpanded @isExpandedByDefault)}}
+>
+  <h2 class="title" data-test-title>
+    <button
+      class={{if this.isExpanded "expanded" "expandable"}}
+      type="button"
+      {{on "click" (set this.isExpanded (not this.isExpanded))}}
+      data-test-expand-collapse
+    >
+      {{@title}}
+    </button>
     {{#if this.showDisplayModeToggle}}
       <button
-        class={{if this.groupByCompetencies "active"}}
+        class={{unless this.isExpanded "disabled" (if this.groupByCompetencies "active")}}
```

Comment:
I'm having trouble reading this one. Is it two expressions or one?
```suggestion
        class="{{unless this.isExpanded "disabled"}} {{if this.groupByCompetencies "active"}}"
```

Reply:
actually, that's one expression. i merged your suggestion too soon. undoing it...

Reply:
yes, so it's either disabled, or active. but cannot be both, since they both apply background-color styles to this button.

Reply:
Ah, I see it now. If the first is false then the second gets read and if that is true show up. Ok.